ligo/test/Ligo/BaseDAO/Contract.hs

```diff
@@ -0,0 +1,33 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+-- | LIGO version of the contract.
+module Ligo.BaseDAO.Contract
+  ( baseDAOContractLigo
+  ) where
+
+import Universum
+
+import Control.Exception.Uncaught (DisplayExceptionInShow(..))
+import GHC.IO.Unsafe (unsafePerformIO)
+import System.Environment (lookupEnv)
+
+import Michelson.Test.Import
+import Michelson.Typed
+
+import Ligo.BaseDAO.Types
+
+fetchContract :: (KnownT cp, KnownT st) => String -> Contract cp st
+fetchContract envKey = unsafePerformIO . rewrapExc $ do
+  mPath <- lookupEnv envKey
+  let path = mPath
+          ?: error ("Specify path to contract in " <> show envKey <> " var")
+             -- ↑ Note that our working directory will be './ligo/test'
+  (_uc, tc) <- importContract path
+  return $  tc
+  where
+    rewrapExc = handleAny $ throwM . DisplayExceptionInShow
+
+baseDAOContractLigo :: Contract (ToT Parameter) (ToT FullStorage)
+baseDAOContractLigo = fetchContract "BASEDAO_LIGO_PATH"
```

Comment:
Can we use TH like we do in https://github.com/tqtezos/stablecoin? It sounds safer and cleaner. Maybe we should even add a helper to `morley` to import a contract in compile-time.

Reply:
Apparently in this case this makes perfect sense.